### ISSUE TITLE
Regression: No log output for format "%z", "%:z", "%Z" or "%+" 

### ISSUE THREAD

**Issue by timokroeger** [Drakulix/simplelog.rs#49]
A logger configured with any timezone related output format does not work.
No output can be seen on the terminal for a TermLogger.
No output is written to the logfile for a WriteLogger.

Example:
````rust
// Setup logging: Use local time in RFC3339 format
let log_config = LogConfigBuilder::new()
    .set_time_to_local(true)
    .set_time_format_str("%Y-%m-%dT%H:%M:%S%.3f%:z")
    .build();
````

As soon as I remove the last "%:z" the logger works as expected.

Tested version: 0.7.3
Version 0.5 and 0.6 work just fine with those format specifiers.

**Comment by AlexanderThaller** [comment]
Have the same problem with this setup

```rust
fn main() {
    let config = simplelog::ConfigBuilder::new()
        .set_time_format("%+".to_string())
        .build();

    println!("config: {:#?}", config);

    if let Err(err) =
        simplelog::TermLogger::init(opt.log_level, config, simplelog::TerminalMode::Stderr)
    {
        eprintln!("can not initialize logger: {}", err);
        ::std::process::exit(1);
    }

    error!("test");

    info!("test");
}
```

**Comment by Drakulix** [comment]
Fix published as v0.7.4
